nuxeo/templates/pvc.yaml

```diff
@@ -7,21 +7,17 @@ metadata:
 {{- with .Values.persistentVolumeStorage.labels }}
 {{ toYaml . | indent 4 }}
 {{- end }}
-  annotations:
-  {{- if .Values.persistentVolumeStorage.storageClass }}
-    volume.beta.kubernetes.io/storage-class: {{ .Values.persistentVolumeStorage.storageClass | quote }}
-  {{- else }}
-    volume.alpha.kubernetes.io/storage-class: default
-  {{- end }}
 {{- with .Values.persistentVolumeStorage.annotations }}
-{{ toYaml . | indent 4 }}
+  annotations: {{ toYaml . | indent 4 }}
```

Comment:
You need `nindent`, otherwise the following values:
```
persistentVolumeStorage:
  enabled: true
  annotations:
    foo: bar
    jack: joe
```
will fail the chart templating:
```
      annotations:     foo: bar
    jack: joe
```